routing/routing_tests/cross_mwm_connector_test.cpp

```diff
@@ -236,4 +236,76 @@ UNIT_TEST(Serialization)
             {{Segment(mwmId, 10, 1, true /* forward */), kEdgesWeight},
              {Segment(mwmId, 20, 2, true /* forward */), kEdgesWeight}});
 }
+
+UNIT_TEST(WeightsSerialization)
+{
+  size_t constexpr kNumTransitions = 3;
+  std::vector<double> const weights = {
```

Comment:
There's ```using namespace std;``` at the beginning of the file. I propose to remove all the ```std::``` from this cpp